applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -1599,7 +1599,22 @@ static int handle_at_getaddrinfo(enum at_cmd_type cmd_type, const struct at_para
 		if (err) {
 			return err;
 		}
-		err = getaddrinfo(host, NULL, NULL, &result);
+		if (param_count > 2) {
+			/* DNS query with designated address family */
+			struct addrinfo hints = {
+				.ai_family = AF_UNSPEC
+			};
+			err = at_params_int_get(param_list, 2, &hints.ai_family);
```

Comment:
FYI this could've used `at_params_unsigned_int_get()`.

Reply:
`hints.ai_family` is of a type `int`